### DISCUSSION TITLE
为什么会有该项报错，在我尝试安装tailwindcss后依然存在，无法启动本地服务器。

### DISCUSSION THREAD

**Discussion by lym0518** [vuepress-reco/vuepress-theme-reco#50] [Q&A]
Module Error (from ./node_modules/postcss-loader/dist/cjs.js):
Loading PostCSS "tailwindcss" plugin failed: Cannot find module 'tailwindcss'

![image](https://user-images.githubusercontent.com/78287901/163998000-6acae4ef-1c07-47ac-bcfb-1c5498936e9f.png)

**Comment by recoluan** [comment]
同学，将 vuepress 和 reco 主题都升级到最新的 beta 版本，分别是 2.0.0-beta.39 和 2.0.0-beta.5，清除一下 node_modules ，lock 文件，以及 .vuepress 文件夹下的两个缓存文件夹，重新 install 。
